### ISSUE TITLE
역할 분담

### ISSUE THREAD

**Issue by Junduck15** [junehee0205/2020OSS_team12#2]
어떤게 편하신가요?
1. 코딩
2. 위키
3. 알고리즘
4. 내용 정리

**Comment by junehee0205** [comment]
저는 알고리즘과 내용 정리를 하겠습니다!

**Comment by Junduck15** [comment]
그러면 저는 코딩이랑 위키를 하겠습니다

**Comment by Junduck15** [comment]
반 관리 시스템을 학생들의 정보 (이름, 학번, 전공)을 추가하는 CRUD프로젝트 어떤가요?

**Comment by junehee0205** [comment]
저희 학교의 특징을 살려서 반 대신 팀으로 하는 건 어떤가요?

**Comment by Junduck15** [comment]
네 좋습니다. 그러면 일단 기존에 저희가 mini_project에서 하던대로 알고리즘을 사용하여서 헤더파일을 생성해보면 될거같습니다.

**Comment by Junduck15** [comment]
Readinfo
Addinfo
Updateinfo
Deleteinfo
Listinfo
Savedata
Loaddata
searchByName
searchByNum
searchByMajor
이렇게 함수들 사용하면 될거같습니다.
추가할게 있을까요?

**Comment by junehee0205** [comment]
selectmenu 함수가 빠진 것 같습니다!

**Comment by junehee0205** [comment]
selectdatano함수도요!
